modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -134,6 +134,11 @@ vec4 project_position(vec4 position, vec2 position64xyLow) {
   // METER_OFFSETS or IDENTITY
   // Apply model matrix
   vec4 position_modelspace = project_uModelMatrix * position;
+  if (project_uCoordinateSystem == COORDINATE_SYSTEM_IDENTITY) {
+    position_modelspace.xy -= project_uCoordinateOrigin;
+    position_modelspace += project_uModelMatrix * vec4(position64xyLow, 0.0, 0.0);
```

Comment:
Should the last element be a `1`? I.e. should model matrix translations not apply?

Reply:
Model matrix translation is already applied to the high parts of the position.

```
uModelMatrix * vec4(
  position.x + position64xyLow.x,
  position.y + position64xyLow.y,
  position.z,
  1.0
) =
uModelMatrix * vec4(position, 1.0) + uModelMatrix * vec4(position64xyLow, 0.0, 0.0)
```

Reply:
I see. Maybe add a comment about that, that translation must only be be added to the high parts.